### DISCUSSION TITLE
hide subtitle on view expanded?

### DISCUSSION THREAD

**Discussion by Jatin9098** [congthanhng/Expansion-Tile-Group#45] [General]
I have tried to hide the subtitle on view expanded but not working. So my question is if tile contains title, subtitle and its children and on view expanded I want hide the subtitle. Is it achievable?

**Comment by congthanhng** [comment]
hi @Jatin9098 , can you please provide me the record of this issue?

**Comment by Jatin9098** [comment]
Hi @congthanhng In below recoding on click "Read More" then tile will be expanded So can we hide subtitle("Read more" only)? I have checked all the methods/documentation but its not achievable yet. And on click of "Read Less" i.e tile collapsed need to show again "Read More".

https://github.com/user-attachments/assets/c2692cdb-1b85-4e15-8d01-b6620e3a0bf5

**Comment by congthanhng** [comment]
hi @Jatin9098 new version 1.2.5 is coming to pub.dev. On new version, just adding new parameter `isHideSubtitleOnExpanded` == true into your item code. Thanks

**Comment by Jatin9098** [comment]
Thanks @congthanhng now its working perfectly! **Amazing work**
